index3.html

```diff
@@ -0,0 +1,42 @@
+<html>
+    <head>
+        <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&family=Roboto:wght@500&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    </head>
+    <body>
+        <div class="container">
+            <div class="header-container">
+                <div class="header">
+                    <div class="left">
+                        <div class="clock">9:41</div>
+                    </div>
+                    <div class="right">
+                        <div class="icons">
+                            <i class="fa-solid fa-signal"></i>
+                            <i class="fa-solid fa-wifi"></i>
+                            <i class="fa-solid fa-battery-full"></i>
+                        </div>
+                    </div>
+                </div>
+                <nav>
+                    <a href="#" class="logo">My Logo</a>
+                    <div class="menu"><img src="menu.png" width="30" height="30" ></div>
+                </nav>
+            </div>
+            
+            <div class="background">
+                <div class="text-container">
+                    <h1 class="title">I'm Linh Chi. Glad to see you!</h1>
+                    <p class="paragraph">I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+                </div>
+                <div class="connect">LET'S CONNECT</div>
+                <div class="social-media">
+                    <i class="fa-brands fa-twitter"></i>
+                    <i class="fa-brands fa-linkedin-in"></i>
+                    <i class="fa-brands fa-github"></i>
+                </div>
+            </div>
+        </div>
+    </body>
+</html>
```

Comment:
- [ Optional ] Kindly I recommend you to have only one `.html` file, So for best practice of HTML please remove this `index3.html` from your code base.